programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md

```diff
@@ -234,10 +234,30 @@ Onboarding new end users into a sophisticated system like Knative Eventing prese
     - Experience with Figma is a big plus.
     - Experience with Rest APIs and Golang is a plus too.
 - Mentor(s):
-    - Florent Poinsard @fouioui florent@planetscale.com
+    - Florent Poinsard @frouioui florent@planetscale.com
     - Frances Thai @notfelineit frances@planetscale.com
 - Upstream Issue (URL): https://github.com/vitessio/arewefastyet/issues/525
 
+#### Community building and engagement
+
+- Description:
+    - [Vitess](https://vitess.io) is a CNCF project that has been around for a while. It has a strong community of users and contributors. We want to continue growing this community and make sure that everyone feels welcome and included.
+- Expected Outcome:
+    - The mentee is expected to evaluate contributor ladder schemes and rewards and produce a recommendation for the Vitess maintainers.
+    - Once a decision is made, the mentee is expected to implement the decisions from the maintainer team.
+    - The mentee is expected to collect data about Vitess usage from the community and publish the highlights as a blog post.
+    - The mentee is expected to review the [Getting Started docs on the Vitess website](https://vitess.io/docs/20.0/get-started/) and enhance them to improve the onboarding experience.
+    - The mentee is expected to research and recommend marketing opportunities for Vitess. These could be guest blog posts, podcasts, live streams etc.
+- Recommended Skills:
+    - Excellent verbal and written communication skills.
+    - Prior experience participating in an open source community is a plus.
+    - Should be able to install and run Vitess according to the user guides.
+    - Website development skills are a plus.
+- Mentor(s):
+    - Deepthi Sigireddi @deepthi deepthi@planetscale.com
+    - Florent Poinsard @frouioui florent@planetscale.com
+- Upstream Issue (URL): N/A
```

Comment:
Thanks @deepthi @frouioui 

It would be super nice if you can create an issue for this work (or, multiple actually).

I will wait a bit before merging, if you want to do that.

Reply:
+1 it's important to have an upstream issue for candidates to see what they're getting into. It's also helpful as a spot to have conversations about the work.

Reply:
(also, we need an upstream issue to link to when we load the project into the LFX platform)

Reply:
@aliok I have actually created 2 issues, but one is linked into the other, so I've only added the main issue into the proposal.